buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy

```diff
@@ -1244,7 +1237,7 @@ class BeamModulePlugin implements Plugin<Project> {
 
           project.dependencies {
             shadowTestRuntimeClasspath it.project(path: project.path, configuration: "shadowTest")
-            shadowTestRuntimeClasspath it.project(path: project.path, configuration: "compileOnly")
```

Comment:
What is the impact of this change, will this affect the artifacts we publish?

Reply:
This changes shadowTest targets from depending on the compile only classpath of the underlying target to depending on the implementation. If everything runs, it should not affect any artifacts.